Notifications

torbjornjohannsen opened the pull request:
Closes #88

T0rvadaL commented:
- Tagging authors with spaces in their username does not seem to be possible.
- There is a missing "|" between the notifications and logout buttons in the navbar
- The fields in the Notification, NotificationDTO and NotificationModel classes should be capitalized according to C# naming convention (this is merely a suggestion as I might be the only one that cares about this stuff.)
- You put the GetNotifications in the AuthorRepository class. We should have one repository per domain model according to [this](https://github.com/itu-bdsa/lecture_notes/blob/main/sessions/session_13/README_PROJECT.md#refactoring-repositories-and-data-transfer-objects-dto), so i believe it should be in its own NotificationRepository class
- There is a Console.WriteLine method call in the OnGetAsync method in NotificationsModel and in the GetAuthorsNotificationd mdethod in CheepService. Also you added `options.EnableSensitiveDataLogging()` in program.cs Perhaps you used it for debugging and forgot to delete it? Not a big deal either way, just a heads up.

torbjornjohannsen commented:
- Didn't know authors could have spaces in their usernames.  Changed to disallow it and updated tests
- Fixed
- Fixed(Except for Notification, since that would require a whole new migration which seems a bit silly for capitalization) 
- Fixed, was just too lazy to add a new repository for two methods 
- Fixed, wrt options it makes debugging certain problems with the database much much easier, so think we should keep it in development. Not like we have any sensitive data there.
